diff -up kdenlive-20.03.80/src/assets/view/widgets/curves/bezier/beziersplineeditor.cpp.omv~ kdenlive-20.03.80/src/assets/view/widgets/curves/bezier/beziersplineeditor.cpp
--- kdenlive-20.03.80/src/assets/view/widgets/curves/bezier/beziersplineeditor.cpp.omv~	2020-03-23 19:44:02.566949227 +0100
+++ kdenlive-20.03.80/src/assets/view/widgets/curves/bezier/beziersplineeditor.cpp	2020-03-23 19:44:10.712908011 +0100
@@ -24,6 +24,7 @@
 
 #include <QMouseEvent>
 #include <QPainter>
+#include <QPainterPath>
 
 BezierSplineEditor::BezierSplineEditor(QWidget *parent)
     : AbstractCurveWidget(parent)
diff -up kdenlive-20.03.80/src/bin/abstractprojectitem.cpp.omv~ kdenlive-20.03.80/src/bin/abstractprojectitem.cpp
--- kdenlive-20.03.80/src/bin/abstractprojectitem.cpp.omv~	2020-03-23 19:43:21.963154649 +0100
+++ kdenlive-20.03.80/src/bin/abstractprojectitem.cpp	2020-03-23 19:43:27.191128203 +0100
@@ -32,6 +32,7 @@ along with this program.  If not, see <h
 #include "jobs/thumbjob.hpp"
 
 #include <QPainter>
+#include <QPainterPath>
 #include <QVariant>
 #include <utility>
 AbstractProjectItem::AbstractProjectItem(PROJECTITEMTYPE type, QString id, const std::shared_ptr<ProjectItemModel> &model, bool isRoot)
diff -up kdenlive-20.03.80/src/titler/gradientwidget.cpp.omv~ kdenlive-20.03.80/src/titler/gradientwidget.cpp
--- kdenlive-20.03.80/src/titler/gradientwidget.cpp.omv~	2020-03-23 19:45:50.967400617 +0100
+++ kdenlive-20.03.80/src/titler/gradientwidget.cpp	2020-03-23 19:45:55.783376236 +0100
@@ -23,6 +23,7 @@ along with this program.  If not, see <h
 
 #include <QLinearGradient>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPixmap>
 #include <QtMath>
 
