diff -up kdenlive-20.03.90/src/assets/view/widgets/keywordparamwidget.cpp.omv~ kdenlive-20.03.90/src/assets/view/widgets/keywordparamwidget.cpp
--- kdenlive-20.03.90/src/assets/view/widgets/keywordparamwidget.cpp.omv~	2020-04-13 14:05:57.937324987 +0200
+++ kdenlive-20.03.90/src/assets/view/widgets/keywordparamwidget.cpp	2020-04-13 14:06:00.917359419 +0200
@@ -56,11 +56,7 @@ KeywordParamWidget::KeywordParamWidget(s
     connect(lineeditwidget, &QLineEdit::editingFinished, [this]() { 
         emit valueChanged(m_index, lineeditwidget->text(), true);
     });
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(comboboxwidget, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
-#else
-    connect(comboboxwidget, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) ,
-#endif
             [this](int ix) {
             if (ix > 0) {
                 QString comboval = comboboxwidget->itemData(ix).toString();
diff -up kdenlive-20.03.90/src/assets/view/widgets/listparamwidget.cpp.omv~ kdenlive-20.03.90/src/assets/view/widgets/listparamwidget.cpp
--- kdenlive-20.03.90/src/assets/view/widgets/listparamwidget.cpp.omv~	2020-04-13 14:04:59.486648676 +0200
+++ kdenlive-20.03.90/src/assets/view/widgets/listparamwidget.cpp	2020-04-13 14:05:02.857687734 +0200
@@ -46,11 +46,7 @@ ListParamWidget::ListParamWidget(std::sh
 
     // emit the signal of the base class when appropriate
     // The connection is ugly because the signal "currentIndexChanged" is overloaded in QComboBox
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(this->m_list, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
-#else
-    connect(this->m_list, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) ,
-#endif
             [this](int) {
                 emit valueChanged(m_index, m_list->itemData(m_list->currentIndex()).toString(), true);
     });
diff -up kdenlive-20.03.90/src/dialogs/encodingprofilesdialog.cpp.omv~ kdenlive-20.03.90/src/dialogs/encodingprofilesdialog.cpp
--- kdenlive-20.03.90/src/dialogs/encodingprofilesdialog.cpp.omv~	2020-04-13 14:05:38.421099379 +0200
+++ kdenlive-20.03.90/src/dialogs/encodingprofilesdialog.cpp	2020-04-13 14:05:41.008129300 +0200
@@ -46,11 +46,7 @@ EncodingProfilesDialog::EncodingProfiles
 
     m_configFile = new KConfig(QStringLiteral("encodingprofiles.rc"), KConfig::CascadeConfig, QStandardPaths::AppDataLocation);
     profile_type->setCurrentIndex(profileType);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(profile_type, static_cast<void (KComboBox::*)(int)>(&KComboBox::currentIndexChanged), this, &EncodingProfilesDialog::slotLoadProfiles);
-#else
-    connect(profile_type, static_cast<void (KComboBox::*)(int, const QString &)>(&KComboBox::currentIndexChanged) , this, &EncodingProfilesDialog::slotLoadProfiles);
-#endif
     connect(profile_list, &QListWidget::currentRowChanged, this, &EncodingProfilesDialog::slotShowParams);
     connect(button_delete, &QAbstractButton::clicked, this, &EncodingProfilesDialog::slotDeleteProfile);
     connect(button_add, &QAbstractButton::clicked, this, &EncodingProfilesDialog::slotAddProfile);
diff -up kdenlive-20.03.90/src/dialogs/kdenlivesettingsdialog.cpp.omv~ kdenlive-20.03.90/src/dialogs/kdenlivesettingsdialog.cpp
--- kdenlive-20.03.90/src/dialogs/kdenlivesettingsdialog.cpp.omv~	2020-04-13 14:02:35.880978015 +0200
+++ kdenlive-20.03.90/src/dialogs/kdenlivesettingsdialog.cpp	2020-04-13 14:03:04.011306412 +0200
@@ -165,17 +165,9 @@ KdenliveSettingsDialog::KdenliveSettings
             }
         }
     }
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.kcfg_detectedv4ldevices, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.kcfg_detectedv4ldevices, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdatev4lDevice);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.kcfg_v4l_format, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.kcfg_v4l_format, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdatev4lCaptureProfile);
     connect(m_configCapture.config_v4l, &QAbstractButton::clicked, this, &KdenliveSettingsDialog::slotEditVideo4LinuxProfile);
 
@@ -265,24 +257,12 @@ KdenliveSettingsDialog::KdenliveSettings
 
     loadEncodingProfiles();
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configSdl.kcfg_audio_driver, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configSdl.kcfg_audio_driver, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotCheckAlsaDriver);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configSdl.kcfg_audio_backend, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configSdl.kcfg_audio_backend, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotCheckAudioBackend);
     initDevices();
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.kcfg_grab_capture_type, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.kcfg_grab_capture_type, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdateGrabRegionStatus);
 
     slotUpdateGrabRegionStatus();
@@ -297,11 +277,7 @@ KdenliveSettingsDialog::KdenliveSettings
     m_configCapture.decklink_parameters->setVisible(false);
     m_configCapture.decklink_parameters->setMaximumHeight(QFontMetrics(font()).lineSpacing() * 4);
     m_configCapture.decklink_parameters->setPlainText(KdenliveSettings::decklink_parameters());
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.kcfg_decklink_profile, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.kcfg_decklink_profile, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdateDecklinkProfile);
     connect(m_configCapture.decklink_showprofileinfo, &QAbstractButton::clicked, m_configCapture.decklink_parameters, &QWidget::setVisible);
 
@@ -315,11 +291,7 @@ KdenliveSettingsDialog::KdenliveSettings
     act->setData(2);
     connect(act, &QAction::triggered, this, &KdenliveSettingsDialog::slotManageEncodingProfile);
     m_configCapture.v4l_manageprofile->setDefaultAction(act);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.kcfg_v4l_profile, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.kcfg_v4l_profile, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdateV4lProfile);
     connect(m_configCapture.v4l_showprofileinfo, &QAbstractButton::clicked, m_configCapture.v4l_parameters, &QWidget::setVisible);
 
@@ -332,11 +304,7 @@ KdenliveSettingsDialog::KdenliveSettings
     act->setData(3);
     connect(act, &QAction::triggered, this, &KdenliveSettingsDialog::slotManageEncodingProfile);
     m_configCapture.grab_manageprofile->setDefaultAction(act);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.kcfg_grab_profile, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.kcfg_grab_profile, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdateGrabProfile);
     connect(m_configCapture.grab_showprofileinfo, &QAbstractButton::clicked, m_configCapture.grab_parameters, &QWidget::setVisible);
 
@@ -347,11 +315,7 @@ KdenliveSettingsDialog::KdenliveSettings
 
     int channelsIndex = m_configCapture.audiocapturechannels->findData(KdenliveSettings::audiocapturechannels());
     m_configCapture.audiocapturechannels->setCurrentIndex(qMax(channelsIndex, 0));
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.audiocapturechannels, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.audiocapturechannels, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdateAudioCaptureChannels);
 
     // audio capture sample rate
@@ -361,11 +325,7 @@ KdenliveSettingsDialog::KdenliveSettings
 
     int sampleRateIndex = m_configCapture.audiocapturesamplerate->findData(KdenliveSettings::audiocapturesamplerate());
     m_configCapture.audiocapturesamplerate->setCurrentIndex(qMax(sampleRateIndex, 0));
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.audiocapturesamplerate, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configCapture.audiocapturesamplerate, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdateAudioCaptureSampleRate);
 
     m_configCapture.labelNoAudioDevices->setVisible(false);
@@ -375,11 +335,7 @@ KdenliveSettingsDialog::KdenliveSettings
     act->setData(1);
     connect(act, &QAction::triggered, this, &KdenliveSettingsDialog::slotManageEncodingProfile);
     m_configProject.preview_manageprofile->setDefaultAction(act);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configProject.kcfg_preview_profile, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configProject.kcfg_preview_profile, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdatePreviewProfile);
     connect(m_configProject.preview_showprofileinfo, &QAbstractButton::clicked, m_configProject.previewparams, &QWidget::setVisible);
     m_configProject.previewparams->setVisible(false);
@@ -407,11 +363,7 @@ KdenliveSettingsDialog::KdenliveSettings
     m_configProxy.proxy_manageprofile->setDefaultAction(act);
 
     connect(m_configProxy.proxy_showprofileinfo, &QAbstractButton::clicked, m_configProxy.proxyparams, &QWidget::setVisible);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configProxy.kcfg_proxy_profile, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this,
-#else
-    connect(m_configProxy.kcfg_proxy_profile, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this,
-#endif
             &KdenliveSettingsDialog::slotUpdateProxyProfile);
 
     slotUpdateProxyProfile(-1);
@@ -483,11 +435,7 @@ bool KdenliveSettingsDialog::initAudioRe
     m_configCapture.labelNoAudioDevices->setVisible(audioDevices.empty());
 
     m_configCapture.kcfg_defaultaudiocapture->addItems(audioDevices);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_configCapture.kcfg_defaultaudiocapture, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), [&]() {
-#else
-    connect(m_configCapture.kcfg_defaultaudiocapture, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , [&]() {
-#endif
         QString currentDevice = m_configCapture.kcfg_defaultaudiocapture->currentText();
         KdenliveSettings::setDefaultaudiocapture(currentDevice);
     });
diff -up kdenlive-20.03.90/src/dialogs/renderwidget.cpp.omv~ kdenlive-20.03.90/src/dialogs/renderwidget.cpp
--- kdenlive-20.03.90/src/dialogs/renderwidget.cpp.omv~	2020-04-13 14:01:51.231455480 +0200
+++ kdenlive-20.03.90/src/dialogs/renderwidget.cpp	2020-04-13 14:01:57.524529229 +0200
@@ -361,11 +361,7 @@ RenderWidget::RenderWidget(bool enablePr
         m_view.parallel_process->setEnabled(false);
     }
     m_view.field_order->setEnabled(false);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_view.scanning_list, QOverload<int>::of(&QComboBox::currentIndexChanged), [this](int index) { m_view.field_order->setEnabled(index == 2); });
-#else
-    connect(m_view.scanning_list, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), [this](int index) { m_view.field_order->setEnabled(index == 2); });
-#endif
     refreshView();
     focusFirstVisibleItem();
     adjustSize();
diff -up kdenlive-20.03.90/src/dialogs/titletemplatedialog.cpp.omv~ kdenlive-20.03.90/src/dialogs/titletemplatedialog.cpp
--- kdenlive-20.03.90/src/dialogs/titletemplatedialog.cpp.omv~	2020-04-13 14:05:47.800207830 +0200
+++ kdenlive-20.03.90/src/dialogs/titletemplatedialog.cpp	2020-04-13 14:05:50.696241304 +0200
@@ -63,11 +63,7 @@ TitleTemplateDialog::TitleTemplateDialog
     }
     const QStringList mimeTypeFilters = {QStringLiteral("application/x-kdenlivetitle")};
     m_view.template_list->setMimeTypeFilters(mimeTypeFilters);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_view.template_list->comboBox(), static_cast<void (KComboBox::*)(int)>(&KComboBox::currentIndexChanged), this,
-#else
-    connect(m_view.template_list->comboBox(), static_cast<void (KComboBox::*)(int, const QString &)>(&KComboBox::currentIndexChanged) , this,
-#endif
             &TitleTemplateDialog::updatePreview);
     updatePreview();
 }
diff -up kdenlive-20.03.90/src/dvdwizard/dvdwizardchapters.cpp.omv~ kdenlive-20.03.90/src/dvdwizard/dvdwizardchapters.cpp
--- kdenlive-20.03.90/src/dvdwizard/dvdwizardchapters.cpp.omv~	2020-04-13 14:04:09.225065526 +0200
+++ kdenlive-20.03.90/src/dvdwizard/dvdwizardchapters.cpp	2020-04-13 14:04:13.315113040 +0200
@@ -27,11 +27,7 @@ DvdWizardChapters::DvdWizardChapters(Mon
 
 {
     m_view.setupUi(this);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_view.vob_list, static_cast<void (KComboBox::*)(int)>(&KComboBox::currentIndexChanged), this, &DvdWizardChapters::slotUpdateChaptersList);
-#else
-    connect(m_view.vob_list, static_cast<void (KComboBox::*)(int, const QString &)>(&KComboBox::currentIndexChanged) , this, &DvdWizardChapters::slotUpdateChaptersList);
-#endif
     connect(m_view.button_add, &QAbstractButton::clicked, this, &DvdWizardChapters::slotAddChapter);
     connect(m_view.button_delete, &QAbstractButton::clicked, this, &DvdWizardChapters::slotRemoveChapter);
     connect(m_view.chapters_list, &QListWidget::itemSelectionChanged, this, &DvdWizardChapters::slotGoToChapter);
diff -up kdenlive-20.03.90/src/dvdwizard/dvdwizardmenu.cpp.omv~ kdenlive-20.03.90/src/dvdwizard/dvdwizardmenu.cpp
--- kdenlive-20.03.90/src/dvdwizard/dvdwizardmenu.cpp.omv~	2020-04-13 14:05:12.986805051 +0200
+++ kdenlive-20.03.90/src/dvdwizard/dvdwizardmenu.cpp	2020-04-13 14:05:15.592835224 +0200
@@ -238,11 +238,7 @@ DvdWizardMenu::DvdWizardMenu(DVDFORMAT f
     connect(m_view.background_image, &KUrlRequester::textChanged, this, &DvdWizardMenu::buildImage);
     connect(m_view.background_color, &KColorButton::changed, this, &DvdWizardMenu::buildColor);
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_view.background_list, static_cast<void (KComboBox::*)(int)>(&KComboBox::currentIndexChanged), this, &DvdWizardMenu::checkBackgroundType);
-#else
-    connect(m_view.background_list, static_cast<void (KComboBox::*)(int, const QString &)>(&KComboBox::currentIndexChanged) , this, &DvdWizardMenu::checkBackgroundType);
-#endif
 
     connect(m_view.target_list, static_cast<void (KComboBox::*)(int)>(&KComboBox::activated), this, &DvdWizardMenu::setButtonTarget);
     connect(m_view.back_to_menu, &QAbstractButton::toggled, this, &DvdWizardMenu::setBackToMenu);
diff -up kdenlive-20.03.90/src/mainwindow.cpp.omv~ kdenlive-20.03.90/src/mainwindow.cpp
--- kdenlive-20.03.90/src/mainwindow.cpp.omv~	2020-04-13 14:04:45.350484817 +0200
+++ kdenlive-20.03.90/src/mainwindow.cpp	2020-04-13 14:04:47.949514953 +0200
@@ -291,11 +291,7 @@ void MainWindow::init()
         for (int ix = 0; ix < screens; ix++) {
             screenCombo->addItem(i18n("Monitor %1", ix));
         }
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
         connect(screenCombo, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), m_clipMonitor, &Monitor::slotSetScreen);
-#else
-        connect(screenCombo, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , m_clipMonitor, &Monitor::slotSetScreen);
-#endif
         recToolbar->addWidget(screenCombo);
         // Update screen grab monitor choice in case we changed from fullscreen
         screenCombo->setEnabled(KdenliveSettings::grab_capture_type() == 0);
diff -up kdenlive-20.03.90/src/mltcontroller/clippropertiescontroller.cpp.omv~ kdenlive-20.03.90/src/mltcontroller/clippropertiescontroller.cpp
--- kdenlive-20.03.90/src/mltcontroller/clippropertiescontroller.cpp.omv~	2020-04-13 14:04:23.021225748 +0200
+++ kdenlive-20.03.90/src/mltcontroller/clippropertiescontroller.cpp	2020-04-13 14:04:33.435346612 +0200
@@ -478,11 +478,7 @@ ClipPropertiesController::ClipProperties
         auto *combo = new QComboBox(this);
         combo->addItem(i18n("Interlaced"), 0);
         combo->addItem(i18n("Progressive"), 1);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
         connect(combo, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &ClipPropertiesController::slotComboValueChanged);
-#else
-        connect(combo, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &ClipPropertiesController::slotComboValueChanged);
-#endif
         combo->setObjectName(QStringLiteral("force_progressive_value"));
         if (!force_prog.isEmpty()) {
             combo->setCurrentIndex(force_prog.toInt());
@@ -504,11 +500,7 @@ ClipPropertiesController::ClipProperties
         combo = new QComboBox(this);
         combo->addItem(i18n("Bottom first"), 0);
         combo->addItem(i18n("Top first"), 1);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
         connect(combo, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &ClipPropertiesController::slotComboValueChanged);
-#else
-        connect(combo, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &ClipPropertiesController::slotComboValueChanged);
-#endif
         combo->setObjectName(QStringLiteral("force_tff_value"));
         if (!force_tff.isEmpty()) {
             combo->setCurrentIndex(force_tff.toInt());
@@ -591,11 +583,7 @@ ClipPropertiesController::ClipProperties
                 emit updateClipProperties(m_id, m_originalProperties, properties);
                 m_originalProperties = properties;
             });
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
             QObject::connect(videoStream, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), [this, videoStream]() {
-#else
-            QObject::connect(videoStream, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , [this, videoStream]() {
-#endif
                 QMap<QString, QString> properties;
                 properties.insert(QStringLiteral("video_index"), QString::number(videoStream->currentData().toInt()));
                 emit updateClipProperties(m_id, m_originalProperties, properties);
@@ -650,11 +638,7 @@ ClipPropertiesController::ClipProperties
                 emit updateClipProperties(m_id, m_originalProperties, properties);
                 m_originalProperties = properties;
             });
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
             QObject::connect(audioStream, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), [this, audioStream]() {
-#else
-            QObject::connect(audioStream, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , [this, audioStream]() {
-#endif
                 QMap<QString, QString> properties;
                 properties.insert(QStringLiteral("audio_index"), QString::number(audioStream->currentData().toInt()));
                 emit updateClipProperties(m_id, m_originalProperties, properties);
@@ -709,11 +693,7 @@ ClipPropertiesController::ClipProperties
             combo->setEnabled(false);
         }
         connect(box, &QAbstractButton::toggled, combo, &QWidget::setEnabled);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
         connect(combo, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &ClipPropertiesController::slotComboValueChanged);
-#else
-        connect(combo, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &ClipPropertiesController::slotComboValueChanged);
-#endif
         hlay->addWidget(box);
         hlay->addWidget(combo);
         vbox->addLayout(hlay);
diff -up kdenlive-20.03.90/src/monitor/recmanager.cpp.omv~ kdenlive-20.03.90/src/monitor/recmanager.cpp
--- kdenlive-20.03.90/src/monitor/recmanager.cpp.omv~	2020-04-13 14:02:13.680718408 +0200
+++ kdenlive-20.03.90/src/monitor/recmanager.cpp	2020-04-13 14:02:20.041792834 +0200
@@ -73,11 +73,7 @@ RecManager::RecManager(Monitor *parent)
     m_audio_device = new QComboBox(parent);
     QStringList audioDevices = pCore->getAudioCaptureDevices();
     m_audio_device->addItems(audioDevices);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_audio_device, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &RecManager::slotAudioDeviceChanged);
-#else
-    connect(m_audio_device, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &RecManager::slotAudioDeviceChanged);
-#endif
     QString selectedDevice = KdenliveSettings::defaultaudiocapture();
     int selectedIndex = m_audio_device->findText(selectedDevice);
     if (!selectedDevice.isNull() && selectedIndex > -1) {
@@ -118,11 +114,7 @@ RecManager::RecManager(Monitor *parent)
     if (selectedIndex > -1) {
         m_device_selector->setCurrentIndex(selectedIndex);
     }
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_device_selector, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &RecManager::slotVideoDeviceChanged);
-#else
-    connect(m_device_selector, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &RecManager::slotVideoDeviceChanged);
-#endif
     m_recToolbar->addWidget(m_device_selector);
 
     QAction *configureRec = m_recToolbar->addAction(QIcon::fromTheme(QStringLiteral("configure")), i18n("Configure Recording"));
diff -up kdenlive-20.03.90/src/project/dialogs/profilewidget.cpp.omv~ kdenlive-20.03.90/src/project/dialogs/profilewidget.cpp
--- kdenlive-20.03.90/src/project/dialogs/profilewidget.cpp.omv~	2020-04-13 14:05:25.214946603 +0200
+++ kdenlive-20.03.90/src/project/dialogs/profilewidget.cpp	2020-04-13 14:05:30.164003868 +0200
@@ -111,11 +111,7 @@ ProfileWidget::ProfileWidget(QWidget *pa
         m_filter->setFilterFps(current > 0, current);
         slotFilterChanged();
     };
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_fpsFilt, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), updateFps);
-#else
-    connect(m_fpsFilt, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , updateFps);
-#endif
     int ix = m_fpsFilt->findText(KdenliveSettings::profile_fps_filter());
     if (ix > -1) {
         m_fpsFilt->setCurrentIndex(ix);
@@ -130,11 +126,7 @@ ProfileWidget::ProfileWidget(QWidget *pa
         m_filter->setFilterInterlaced(current != -1, current == 0);
         slotFilterChanged();
     };
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_scanningFilt, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), updateScanning);
-#else
-    connect(m_scanningFilt, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , updateScanning);
-#endif
 
     ix = m_scanningFilt->findText(KdenliveSettings::profile_scanning_filter());
     if (ix > -1) {
diff -up kdenlive-20.03.90/src/project/dialogs/projectsettings.cpp.omv~ kdenlive-20.03.90/src/project/dialogs/projectsettings.cpp
--- kdenlive-20.03.90/src/project/dialogs/projectsettings.cpp.omv~	2020-04-13 14:03:32.270635728 +0200
+++ kdenlive-20.03.90/src/project/dialogs/projectsettings.cpp	2020-04-13 14:03:37.851700698 +0200
@@ -163,11 +163,7 @@ ProjectSettings::ProjectSettings(Kdenliv
     connect(proxy_manageprofile, &QAbstractButton::clicked, this, &ProjectSettings::slotManageEncodingProfile);
     proxy_profile->setToolTip(i18n("Select default proxy profile"));
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(proxy_profile, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &ProjectSettings::slotUpdateProxyParams);
-#else
-    connect(proxy_profile, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &ProjectSettings::slotUpdateProxyParams);
-#endif
     proxyparams->setVisible(false);
     proxyparams->setMaximumHeight(QFontMetrics(font()).lineSpacing() * 5);
     connect(proxy_showprofileinfo, &QAbstractButton::clicked, proxyparams, &QWidget::setVisible);
@@ -183,11 +179,7 @@ ProjectSettings::ProjectSettings(Kdenliv
     connect(preview_manageprofile, &QAbstractButton::clicked, this, &ProjectSettings::slotManagePreviewProfile);
     preview_profile->setToolTip(i18n("Select default preview profile"));
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(preview_profile, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &ProjectSettings::slotUpdatePreviewParams);
-#else
-    connect(preview_profile, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &ProjectSettings::slotUpdatePreviewParams);
-#endif
     previewparams->setVisible(false);
     previewparams->setMaximumHeight(QFontMetrics(font()).lineSpacing() * 5);
     connect(preview_showprofileinfo, &QAbstractButton::clicked, previewparams, &QWidget::setVisible);
diff -up kdenlive-20.03.90/src/titler/titlewidget.cpp.omv~ kdenlive-20.03.90/src/titler/titlewidget.cpp
--- kdenlive-20.03.90/src/titler/titlewidget.cpp.omv~	2020-04-13 14:03:57.407928187 +0200
+++ kdenlive-20.03.90/src/titler/titlewidget.cpp	2020-04-13 14:04:00.084959306 +0200
@@ -490,11 +490,7 @@ TitleWidget::TitleWidget(const QUrl &url
     graphicsView->scene()->addItem(m_frameImage);
 
     bgBox->setCurrentIndex(KdenliveSettings::titlerbg());
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(bgBox, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), [&] (int ix) {
-#else
-    connect(bgBox, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , [&] (int ix) {
-#endif
         KdenliveSettings::setTitlerbg(ix);
         displayBackgroundFrame();
     });
